.changeset/two-wombats-confess.md

```diff
@@ -0,0 +1,17 @@
+---
+'@clerk/clerk-expo': major
+---
+
+Support for [Expo Web](https://docs.expo.dev/workflow/web/) has been added! You can now build fullstack websites with Expo, React, and Clerk. Utilize [Clerk's components](https://clerk.com/docs/components/overview) to build out your app.
+
+You can access the components from the `/web` subpath import like so:
+
+```tsx
+import { SignUp } from "@clerk/clerk-expo/web";
+
+export default function Page() {
+  return <SignUp />;
+}
+```
+
+**Breaking change:** You need to use Expo 50 or later. The minimum required React Native version was bumped to `0.73`.
```

Comment:
Should we mention the reason why we needed to bump ? Either here or in the docs ?

Reply:
This will be mentioned in the docs as part of release cycles for the SDK